resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesConf.scala

```diff
@@ -117,6 +117,7 @@ private[spark] class KubernetesDriverConf(
 
   override def annotations: Map[String, String] = {
     KubernetesUtils.parsePrefixedKeyValuePairs(sparkConf, KUBERNETES_DRIVER_ANNOTATION_PREFIX)
+      .map(p => (p._1, Utils.substituteAppNExecIds(p._2, appId, "")))
```

Comment:
Sorry, but what I suggested (https://github.com/apache/spark/pull/42600#discussion_r1302137555) is the following style instead of `p => ...`.
```scala
case (k, v) => ...
```

Reply:
Sorry,  i misunderstood `the consistent short style`.

Does the following code conform
```
.map{ case(k, v) => (k, Utils.substituteAppNExecIds(v, appId, executorId)) }
```

Reply:
The following?
```scala
.map { case (k, v) => (k, Utils.substituteAppNExecIds(v, appId, executorId)) }
```

Reply:
Yes, I mean does this format conform to `the consistent short style`?

```
.map { case (k, v) => (k, Utils.substituteAppNExecIds(v, appId, executorId)) }
```